src/adlmidi.cpp

```diff
@@ -112,13 +120,14 @@ ADLMIDI_EXPORT int adl_setNumChips(ADL_MIDIPlayer *device, int numChips)
     else if(play->m_setup.numFourOps < -1)
         play->m_setup.numFourOps = -1;
 
-    if(!play->m_synth.setupLocked())
+    OPL3 &synth = *play->m_synth;
```

Comment:
Please make a typedef to unify with OPNMIDI when backporting changes, like I made with MIDI Player class :wink:

Reply:
There doesn't exist any currently, is this correct? I have to make it?

Reply:
I wasn't made any typedefs for Synth as I never used it's name in this file, but that you have made just now is good now :fox_face: :+1: